adapter/memQuota/memQuota.go

```diff
@@ -150,21 +155,16 @@ func (mq *memQuota) Close() error {
 	return nil
 }
 
-// getCurrentTicks returns the number of ticks since January 1st 1970 UTC
-func getCurrentTicks() int64 {
-	return time.Now().UnixNano() / nanosPerTick
-}
-
-func (mq *memQuota) Alloc(args adapter.QuotaArgs) (int64, error) {
+func (mq *memQuota) Alloc(args adapter.QuotaArgs) (int64, time.Duration, error) {
```

Comment:
makes me wonder if we want a type like `QuotaResults` in case we decide we later want to have something more than the amount and valid duration in returned results.

so something like: `func Alloc(args adapter.QuotaArgs) (adapter.QuotaResults, error)`